### ISSUE TITLE
DataTables warning: Table id=example1 - Ajax Error

### ISSUE THREAD

**Issue by juanlock** [dwijitsolutions/laraadmin#91]
I just create one module and when i try to login for list records,, i see this message and don't see any record

DataTables warning:table id=example1 - Ajax error. For more
information about this error, please see http://datatables.net/tn/7

other information, I can see form for add new record, and confirmed that it save it on database.
![2016-10-31_15-46-08](https://cloud.githubusercontent.com/assets/7864172/19869131/6148d180-9f81-11e6-9050-63675ac67d2c.jpg)
![2016-10-31_15-46-29](https://cloud.githubusercontent.com/assets/7864172/19869133/614d3e1e-9f81-11e6-85f3-585908ad2fc8.jpg)
![2016-10-31_15-46-56](https://cloud.githubusercontent.com/assets/7864172/19869130/614862b8-9f81-11e6-85f2-1439c6322ef9.jpg)
![2016-10-31_15-47-30](https://cloud.githubusercontent.com/assets/7864172/19869132/614a533e-9f81-11e6-9c42-5368f4dbaed4.jpg)

**Comment by gdbhosale** [comment]
Can you please run the same on chrome and check network under inspector for data table request. You will see error. Please post the error.

Regards

**Comment by slozano95** [comment]
@juanlock please check @gdbhosale answer, maybe you'll find something like this
![captura de pantalla 2016-11-01 a las 8 14 52 a m](https://cloud.githubusercontent.com/assets/5178713/19890902/7b832a52-a00b-11e6-83ef-08ae018748d4.png)

Now please copy this url in to a new tab on your browser, if you have 'development' and debug true on you laravel app you should see the error, in my case happened because I updated the module but not the migration (SQLSTATE ERROR)

**Comment by juanlock** [comment]
Thanks you.. yes is same message that Slozano95
http://panel2017.stratoinc.com/PanelMis/public/admin/logincontrol_dt_ajax?draw=1&columns%5B0%5D%5Bdata%5D=0&columns%5B0%5D%5Bname%5D=&columns%5B0%5D%5Bsearchable%5D=true&columns%5B0%5D%5Borderable%5D=true&columns%5B0%5D%5Bsearch%5D%5Bvalue%5D=&columns%5B0%5D%5Bsearch%5D%5Bregex%5D=false&columns%5B1%5D%5Bdata%5D=1&columns%5B1%5D%5Bname%5D=&columns%5B1%5D%5Bsearchable%5D=true&columns%5B1%5D%5Borderable%5D=true&columns%5B1%5D%5Bsearch%5D%5Bvalue%5D=&columns%5B1%5D%5Bsearch%5D%5Bregex%5D=false&columns%5B2%5D%5Bdata%5D=2&columns%5B2%5D%5Bname%5D=&columns%5B2%5D%5Bsearchable%5D=true&columns%5B2%5D%5Borderable%5D=true&columns%5B2%5D%5Bsearch%5D%5Bvalue%5D=&columns%5B2%5D%5Bsearch%5D%5Bregex%5D=false&columns%5B3%5D%5Bdata%5D=3&columns%5B3%5D%5Bname%5D=&columns%5B3%5D%5Bsearchable%5D=true&columns%5B3%5D%5Borderable%5D=true&columns%5B3%5D%5Bsearch%5D%5Bvalue%5D=&columns%5B3%5D%5Bsearch%5D%5Bregex%5D=false&columns%5B4%5D%5Bdata%5D=4&columns%5B4%5D%5Bname%5D=&columns%5B4%5D%5Bsearchable%5D=true&columns%5B4%5D%5Borderable%5D=true&columns%5B4%5D%5Bsearch%5D%5Bvalue%5D=&columns%5B4%5D%5Bsearch%5D%5Bregex%5D=false&columns%5B5%5D%5Bdata%5D=5&columns%5B5%5D%5Bname%5D=&columns%5B5%5D%5Bsearchable%5D=true&columns%5B5%5D%5Borderable%5D=true&columns%5B5%5D%5Bsearch%5D%5Bvalue%5D=&columns%5B5%5D%5Bsearch%5D%5Bregex%5D=false&columns%5B6%5D%5Bdata%5D=6&columns%5B6%5D%5Bname%5D=&columns%5B6%5D%5Bsearchable%5D=true&columns%5B6%5D%5Borderable%5D=true&columns%5B6%5D%5Bsearch%5D%5Bvalue%5D=&columns%5B6%5D%5Bsearch%5D%5Bregex%5D=false&columns%5B7%5D%5Bdata%5D=7&columns%5B7%5D%5Bname%5D=&columns%5B7%5D%5Bsearchable%5D=true&columns%5B7%5D%5Borderable%5D=true&columns%5B7%5D%5Bsearch%5D%5Bvalue%5D=&columns%5B7%5D%5Bsearch%5D%5Bregex%5D=false&columns%5B8%5D%5Bdata%5D=8&columns%5B8%5D%5Bname%5D=&columns%5B8%5D%5Bsearchable%5D=true&columns%5B8%5D%5Borderable%5D=true&columns%5B8%5D%5Bsearch%5D%5Bvalue%5D=&columns%5B8%5D%5Bsearch%5D%5Bregex%5D=false&columns%5B9%5D%5Bdata%5D=9&columns%5B9%5D%5Bname%5D=&columns%5B9%5D%5Bsearchable%5D=true&columns%5B9%5D%5Borderable%5D=false&columns%5B9%5D%5Bsearch%5D%5Bvalue%5D=&columns%5B9%5D%5Bsearch%5D%5Bregex%5D=false&order%5B0%5D%5Bcolumn%5D=0&order%5B0%5D%5Bdir%5D=asc&start=0&length=10&search%5Bvalue%5D=&search%5Bregex%5D=false&_=1478027301973

y changed the .env but cannot get see error,
![2016-11-01_15-10-32](https://cloud.githubusercontent.com/assets/7864172/19903618/a4375bd2-a045-11e6-999b-968dab237f6a.jpg)
![2016-11-01_15-09-26](https://cloud.githubusercontent.com/assets/7864172/19903619/a696b8c8-a045-11e6-9f18-3aefb964081a.jpg)

**Comment by slozano95** [comment]
@juanlock please check those settings on config/app.php too,
If you access that link you might see the error

**Comment by juanlock** [comment]
Thanks Santiago, I check on both sides (similar , developer, and True) but don't have luck  :-(. it possible that I'm put something wrong.

But is fixed now.

I have iis 7 / php so I tested from server directly and can see the error 
The error was HTTP Error 404.15 .. The request Filtering module is configure to deny a request......

so only I modified web.config on DocRoot and fixed.

Thanks and appreciate your help.

y put the changes maybe anyone help it.
![2016-11-01_16-21-11](https://cloud.githubusercontent.com/assets/7864172/19906205/3708211c-a050-11e6-92f7-9332f289c634.jpg)
![2016-11-01_16-21-23](https://cloud.githubusercontent.com/assets/7864172/19906203/37066566-a050-11e6-8b5c-b3e0d9d3cdd9.jpg)
![2016-11-01_16-22-03](https://cloud.githubusercontent.com/assets/7864172/19906204/3706c02e-a050-11e6-88d4-1d676265c42c.jpg)
